src/gui/tray/UserModel.h

```diff
@@ -53,6 +59,7 @@ class User : public QObject
     void serverHasTalkChanged();
     void avatarChanged();
     void accountStateChanged(int state);
+    void userStatusChanged();
```

Comment:
@camilasan why do we need *userStatus*Changed? Rest of the fields are named without *user* as prefix, since the class is called `User` anyway.